src/logic/map_objects/findimmovable.h

```diff
@@ -159,6 +159,15 @@ struct FindFlagOf {
 	const FindImmovable finder;
 };
 
+struct FindFlagWithPlayersWarehouse {
+	explicit FindFlagWithPlayersWarehouse(const Player& owner) : owner_(owner) {
+	}
+	[[nodiscard]] bool accept(const BaseImmovable& imm) const;
+
+private:
```

Comment:
would it make sense to make the similar variables of the other structs private as well?